pgmpy/tests/test_readwrite/test_XMLBIF.py

```diff
@@ -173,6 +187,8 @@ def test_get_values(self):
                                                 [0.85]]),
                         'hear_bark': np.array([[0.7, 0.01],
                                                [0.3, 0.99]]),
+                        'kid': np.array([[0.3],
+                                        [0.7]]),
```

Comment:
@lohani2280 PEP8. The second line should be aligned with the second opening bracket of the last line. In short need one extra space on this line.

Reply:
@ankurankan Yeah i'll update them. I have added a new commit just now. Please review that one too.